src/Main.js

```diff
@@ -4,30 +4,31 @@ import ExtendedView from './ExtendedView'
 import List from './List'
 import movieData from './mockData'
 import './Main.css'
+import {getAllMovies, fetchSingleMoviePoster} from './apiCalls'
```

Comment:
Love this. I was working on a second class based component that would make the fetch and hold this data in state, but in the end I do think that this is more efficient.